src/main/java/seedu/application/ui/InterviewCard.java

```diff
@@ -27,9 +27,12 @@ public class InterviewCard extends UiPart<Region> {
     @FXML
     private Label id;
     @FXML
-    private Label company;
+    private Label companyAndPosition;
+    /*
     @FXML
     private Label position;
+
+     */
```

Comment:
Is this intentionally commented out or can it be deleted?

Reply:
Ah thanks for pointing out XD they should be deleted